src/schema/meeting/meetings.ts

```diff
@@ -19,19 +19,64 @@ export const Meeting = objectType({
                 return user;
             },
         });
-        t.list.field('votations', { type: Votation });
+        t.list.field('votations', {
+            type: Votation,
+            resolve: async (source, __, ctx) => {
+                const { id } = source as MeetingType;
+                const votation = await ctx.prisma.votation.findMany({ where: { meetingId: id } });
+                return votation;
+            },
+        });
         t.nonNull.field('status', { type: Status });
+        t.nonNull.field('participants', {
+            type: list(Participant),
+            resolve: async (source, __, ctx) => {
+                const { id } = source as MeetingType;
+                const participants = await ctx.prisma.participant.findMany({
+                    where: {
+                        meetingId: id,
+                    },
+                });
+                return participants;
+            },
+        });
     },
 });
 export const MeetingQuery = extendType({
     type: 'Query',
     definition: (t) => {
-        t.nonNull.field('meetings', {
+        t.nonNull.field('meetings_for_user', {
             // Also possible to write `type: list(Meeting)` and remove it from the `t` part
             type: list(Meeting),
             resolve: async (_, __, ctx) => {
-                const meetings = await ctx.prisma.meeting.findMany();
-                return meetings;
+                const meetingForUser = await ctx.prisma.participant.findMany({
```

Comment:
Rather define fieldResolvers for participants than in a query like this. This is important or else we will get hit by the [N+1 problem](https://medium.com/the-marcy-lab-school/what-is-the-n-1-problem-in-graphql-dd4921cb3c1a). This means that mettings are always fetched, even if we dont request the field